### DISCUSSION TITLE
能不能给文档加个配置好的config.js的范本

### DISCUSSION THREAD

**Discussion by EnderAvaritia** [DIYgod/RSSHub#10500] [Q&A] [answered]
### 这是一个什么样的功能？

希望能给个配置好的config.js的范本

### 这个功能可以解决什么问题？

现在配置服务，完全不知道我不能使用是因为填的是格式错误还是啥，应该填啥格式我也不知道……
如果有个范本的话多少知道点。

### 这不是重复的功能请求

- [X] 我已经搜索了[现有 issue](https://github.com/DIYgod/RSSHub/issues)，以确保这项功能尚未被请求。

**Comment by TonyRL** [comment, marked as answer]
You should **NOT** edit `lib/config.js` directly. You should provide the value through environment variable. The format varies as it depends on how you deploy your instance.

**Comment by EnderAvaritia** [comment]
> You should **NOT** edit `lib/config.js` directly. You should provide the value through environment variable. The format varies as it depends on how you deploy your instance.

![image](https://user-images.githubusercontent.com/84321282/184927795-b70c79b5-f6ef-4e89-81bb-e44c47db6998.png)
it seems i have trouble for my  sbility of understanding it can't work (running in vercel)

**Reply by TonyRL** [reply]
Since you're using Vercel, you can follow https://vercel.com/docs/concepts/projects/environment-variables

**Reply by EnderAvaritia** [reply]
![image](https://user-images.githubusercontent.com/84321282/184938242-5faca425-7da8-44a0-a728-9f34923ff468.png)

it seems still can't work

**Reply by TonyRL** [reply]
Wrong path means you entered the wrong route path

**Reply by EnderAvaritia** [reply]
I open the page by click the link in the official document and only change the beginning of the link
![image](https://user-images.githubusercontent.com/84321282/185140372-505ec007-d0ee-41cc-aba2-52e1ffc18cbb.png)
and it changed today
![image](https://user-images.githubusercontent.com/84321282/185140475-30a83fe3-c765-4e70-9f38-402b9d0d9a65.png)

**Reply by TonyRL** [reply]
Again, wrong path means you entered the wrong route path. Either the path does not exist or you copied a wrong stuff.
After you added a new env in Vercel, you may need to redeploy it as the lower screenshot shows RSSHub does not have this env configured.

**Reply by EnderAvaritia** [reply]
I only replace the beginning of the address without changing another tings 
and I'm sure I have  redeploy the project 
I also refresh the token I filled in
